### PR TITLE
altered styling of project page and removed search filters

### PR DESCRIPTION
**Pull request by attali-david**
#15

### CODE REVIEW

**Comment by lianilychee** [comment]
We can merge once the H2 fix requests get resolved. The other two are minor comments.

**Review by lianilychee — CHANGES_REQUESTED** [review]

File: components/completedProjectList.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import ProjectCard from './ProjectCard'
+import { SimpleGrid, Text, Link } from '@chakra-ui/react'
+import { ProjectListProp } from '../types/Projects'
+
+function CompletedProjectList({ projects }: ProjectListProp) {
+  return (
+    <>
+      <Text>Completed Work</Text>
```
[the remark this answers is not in the document]
Comment (attali-david) [reply components/completedProjectList.tsx:9]: Yes! I held off on styling until I could get confirmation the layout was what you envisioned. I'll fix the styling now.

File: components/completedProjectList.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      <Text>
+        Here are a few of our complete work. For all of our projects visit our
```
[the remark this answers is not in the document]
Comment (attali-david) [reply components/completedProjectList.tsx:11]: This was an oversight on my part. I think either works.

File: components/completedProjectList.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <Link> Github</Link>.
```
[the remark this answers is not in the document]
Comment (newswim) [reply components/completedProjectList.tsx:12]: I would tend to call it a "GitHub Repo", but "GitHub Page" might be a little more approachable (and more accurate than just saying, "Our GitHub".

**Comment by newswim** [comment]
@attali-david since we're no longer using it, you can go ahead and remove `'chakra-react-select'` for the project dependencies.

File: components/completedProjectList.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (lianilychee) [reply components/completedProjectList.tsx:12]: Thanks! Already merged this PR, but referenced you comment in a new issue

**Pull request merged**